src/configure/utilities/templateConverter.ts

```diff
@@ -0,0 +1,106 @@
+const templateParametersRegex = /(inputs|variables|assets|secrets)\.\w+/g;
+const templateEnclosedParametersRegex = /{{{?\s*(inputs|variables|assets|secrets)\.\w+\s*}?}}/g;
+const mustacheHelperRegex = /{?{{\s*#(\w+)(.*?)({?{{\s*\/\1)}}}?/;
```

Comment:
I could not understand this completely. Will catch you for this sometime.

Reply:
sure.